py/README.md

```diff
@@ -0,0 +1,130 @@
+# Braintrust Proxy Python Client
+
+The `braintrust-proxy` package provides Python types, Pydantic models, and helper functions to simplify interaction with the Braintrust AI proxy, particularly enhancing compatibility with the OpenAI Python client.
+
+## Installation
+
+You can install the package using `uv` (recommended) or `pip`:
+
+Using `uv`:
+
+```bash
+uv pip install braintrust-proxy
+```
+
+Using `pip`:
+
+```bash
+pip install braintrust-proxy
+```
+
+## Usage
+
+The package provides enhanced types and helper functions, such as `as_openai_chat_message_param`, to work with messages that might include proxy-specific extensions (like `reasoning`) while maintaining compatibility with standard OpenAI client inputs.
+
+```python
+import os
+from openai import OpenAI
+import json
+from braintrust_proxy import as_openai_chat_message_param
+
+# Initialize the OpenAI client pointing to the Braintrust proxy
+# Ensure BRAINTRUST_API_URL and BRAINTRUST_API_KEY environment variables are set
+client = OpenAI(
+    # if self hosting, otherwise use the hosted API
+    base_url=f"{os.getenv('BRAINTRUST_API_URL') or 'https://api.braintrust.dev'}/v1/proxy",
+    api_key=os.getenv("BRAINTRUST_API_KEY"),
+)
+
+# Example using the helper function with a message that includes 'reasoning'
+response = client.chat.completions.create(
+    model="claude-3-7-sonnet-latest",
+    reasoning_effort=None,
+    stream=False,
+    messages=[
+        {
+            "role": "user",
+            "content": "How many rs in 'ferrocarril'",
+        },
+        as_openai_chat_message_param(
```

Comment:
I don't immediately understand the benefit of this. Could you be more specific in the docs?